scan/lib/scan/options.rb

```diff
@@ -193,9 +194,18 @@ def self.available_options
                                      optional: true),
         FastlaneCore::ConfigItem.new(key: :suppress_xcode_output,
                                      env_name: "SCAN_SUPPRESS_XCODE_OUTPUT",
-                                     description: "Suppress the output of xcodebuild to stdout. Output is still saved in buildlog_path",
+                                     description: "Suppress the output of xcodebuild to stdout. Takes precedence over `enabled_xcode_output`. Output is still saved in buildlog_path",
                                      optional: true,
                                      is_string: false),
```

Comment:
Yeah, I noticed some info about the deprecation of `is_string` but thought that acting on it is not in the scope of this PR. Should it be?

Reply:
No, there is a separate PR that will take care of this. When that one is merged, this will also trigger rubocop to complain - so better get this merged before the other one ;)